.circleci/config.yml

```diff
@@ -4,10 +4,56 @@ orbs:
   macos: circleci/macos@2
   flutter: circleci/flutter@1
   codecov: codecov/codecov@3
+  local-android:
+    orbs:
+      android: circleci/android@2
+    commands:
+      prepare-config:
+        description: "Prepares the environment configuration files"
+        steps:
+          - run:
+              name: Prepare Environment Configuration
+              command: |
+                cp .env.example .env
+                sed -i "s/YOUR_AUTH0_DOMAIN/$AUTH0_DOMAIN/" .env
+                sed -i "s/YOUR_AUTH0_CLIENT_ID/$AUTH0_CLIENT_ID/" .env
+                sed -i "s/YOUR_AUTH0_CUSTOM_SCHEME/$AUTH0_CUSTOM_SCHEME/" .env
+
+                mv android/app/src/main/res/values/strings.xml.example android/app/src/main/res/values/strings.xml
+                sed -i "s/YOUR_AUTH0_DOMAIN/$AUTH0_DOMAIN/" android/app/src/main/res/values/strings.xml
+
+                mv android/local.properties.ci android/local.properties
+              working_directory: ./auth0_flutter/example
+      run-smoke-tests:
+        description: "Runs the Android smoke tests"
+        parameters:
+          android_simulator:
+            type: string
+        steps:
+          - android/accept-licenses
+          - android/create-avd:
+              avd-name: auth0_sample_avd
+              system-image: << parameters.android_simulator >>
+              install: true
+          - android/start-emulator:
+              avd-name: auth0_sample_avd
+              no-window: true
+              restore-gradle-cache-prefix: v1a
+              post-emulator-launch-assemble-command: ""
+              post-emulator-wait-steps:
+                - run:
+                    name: Disable Chrome welcome prompt
+                    command: adb shell 'echo "chrome --disable-fre --no-default-browser-check --no-first-run" > /data/local/tmp/chrome-command-line'
+                - run:
+                    name: Enable ADB root
+                    command: adb root
+          - android/run-tests:
```

Comment:
As mentioned previously in Slack, please make sure to not run the smoke tests for PRs from forks.

Reply:
Do you have an example of what that config looks like?

Reply:
Yup https://github.com/auth0/nextjs-auth0/blob/main/.circleci/config.yml#L30

Reply:
I also included a screenshot with a red-squared example in yesterday's Slack message.

Reply:
There's also another example in this same file: https://github.com/auth0/auth0-flutter/blob/main/.circleci/config.yml#L123

Reply:
Will need to see how we address that here, as smoke tests isnt just a single step, it's an entire job.
Assumably, we need to do something comparable, just slightly different.

Will look into it, thanks.